paimon-core/src/main/java/org/apache/paimon/AbstractFileStore.java

```diff
@@ -60,6 +63,8 @@ public abstract class AbstractFileStore<T> implements FileStore<T> {
 
     @Nullable private final SegmentsCache<String> writeManifestCache;
 
+    private ExecutorService lazyIoExecutor;
```

Comment:
This `IoExecutor` is created in here, but passed to new classes. And closed in new classes.
Its lifetime is weird...

Can we just use `FileUtils.COMMON_IO_FORK_JOIN_POOL`?

Reply:
Resolved.